pkg/windows/build.bat

```diff
@@ -89,7 +89,7 @@ if Defined x (
 if %Python%==2 (
     Set "PyDir=C:\Python27"
 ) else (
-    Set "PyDir=C:\Program Files\Python35"
+    Set "PyDir=C:\Python35"
```

Comment:
I obviously don't know this code well but does this imply that if a person upgrades to this version from one previous that they will have two installations of Python 3.5?

Reply:
if they run the build script and don't run the clean script... then they upgrade to this one... then yes.
This is the script we use for building salt. I wrote it, not intending it to run on a system where they already have python installed. After the script runs, there's a clean script that uninstalls everything so that I can build another version.
If we expect the community to be using this on systems with python installed, then maybe we should look at making this something more salt specific like installing python to `C:\Salt-Build\Python27` / `C:\Salt-Build\Python35` to not overwrite an installation of Python in the default location as it is now (`C:\Python27`).

Reply:
Ah, OK. I didn't realize this wasn't something that is typically run. That's fine then. Thanks for the explanation.